app/src/main/java/com/github/se/eventradar/qrCode/QrCodeGenerator.kt

```diff
@@ -0,0 +1,82 @@
+package com.github.se.eventradar.qrCode
+
+import android.graphics.Bitmap
+import android.graphics.Color
+import com.google.firebase.firestore.FirebaseFirestore
+import com.google.firebase.storage.FirebaseStorage
+import com.google.zxing.BarcodeFormat
+import com.google.zxing.EncodeHintType
+import com.google.zxing.qrcode.QRCodeWriter
+import java.io.ByteArrayOutputStream
+import java.util.*
+
+class QrCodeGenerator {
+
+  fun qrCodeProcess(db: FirebaseFirestore, userId: String) {
+    uploadBitmapToFirestore(generateQRCode(userId), userId, db)
+  }
+
+  private fun generateQRCode(userId: String, width: Int = 512, height: Int = 512): Bitmap {
```

Comment:
The issue you linked to this also included tests. You should add a test for these functions.